cachito/web/utils.py

```diff
@@ -63,3 +65,23 @@ def str_to_bool(item):
         return item.lower() in ("true", "1")
     else:
         return False
+
+
+def deep_sort(orig_item):
+    """
+    Return a new element recursively sorted in ascending order.
+
+    :param orig_item: Original dict or list
+    :return: Recursively sorted dict or list according to orig_item
+    """
+    if isinstance(orig_item, list):
+        sorted_item = [deep_sort(item) for item in orig_item]
+        if len(sorted_item) and isinstance(sorted_item[0], OrderedDict):
+            sorted_item = sorted(sorted_item, key=lambda ordered_dict: ordered_dict.items())
+        else:
+            sorted_item = sorted(sorted_item)
+    elif isinstance(orig_item, dict):
+        sorted_item = OrderedDict(sorted({k: deep_sort(v) for k, v in orig_item.items()}.items()))
+    else:
+        sorted_item = orig_item
```

Comment:
Shouldn't this throw an error if the item is of not type `list` or `dict` ?

Reply:
@nirzari If it is not `list` or `dict` we should not raise an error. For example: 
`deep_sort([1]) ->> deep_sort(1)`

It can be a string, for example.

Reply:
Thank you @chmeliik. 
@nirzari Sorry for not clear description. I changed it now, PTAL.

Reply:
Thanks for explaining. I guess I was just confused by the description earlier.